gui/dig.lua

```diff
@@ -763,12 +763,20 @@ function Dig:getDesignation(x, y, z)
         if z == 0 then
             return stairs_bottom_type == "auto" and "u" or stairs_bottom_type
         elseif z == math.abs(self:get_bounds().z1 - self:get_bounds().z2) then
-            local tile_type = dfhack.maps.getTileType(self:get_bounds().x1 + x, self:get_bounds().y1 + y, z)
+            local tile_type = dfhack.maps.getTileType(self:get_bounds().x1 + x, self:get_bounds().y1 + y, self:get_bounds().z1 + z)
             local tile_shape = tile_type ~= nil and tile_attrs[tile_type].shape or nil
+            local designation = dfhack.maps.getTileFlags(xyz2pos(self:get_bounds().x1 + x, self:get_bounds().y1 + y, self:get_bounds().z1 + z))
```

Comment:
you could pull the coordinates out into a `local pos = xyz2pos(self.getBounds()...xyz)` and pass that to getTileType and getTileFlags to clean this up a little bit